src/app/components/GoogleSheetsInput.svelte

```diff
@@ -54,14 +61,23 @@
       gapi.auth2.getAuthInstance().signIn();
     }
     try {
+      // Create a range of the spreadsheet that includes both columns
+      const spreadsheetRange = `${mapDecToColLetters(
+        Math.min(wordColumnInd, countColumnInd)
+      )}1:${mapDecToColLetters(Math.max(wordColumnInd, countColumnInd))}`;
+
       const spreadsheetId = getSpreadsheetId(googleSheetsURL) || "";
       const {
         result: { values },
       } = await gapi.client.sheets.spreadsheets.values.get({
         spreadsheetId: spreadsheetId,
-        range: "A1:B",
+        range: spreadsheetRange,
       });
-      worker.readGoogleSheet(spreadsheetId, values);
+      const settings = {
+        wordColInd: wordColumnInd,
+        countColInd: countColumnInd,
+      };
```

Comment:
Are there any reasons to name these differently? If we make it consistent, we could do
```suggestion
      const settings = {
        wordColumnInd,
        countColumnInd,
      };
```

Reply:
Fair point +1. My reasoning is somewhat subjective. I wanted the UploadSettings property to be wordColInd rather than wordColumnInd, so that way it's more clear to read and type when using settings.wordColInd

The reason why I named it worldColumnInd in the component prop is that I wanted to be more verbose, as it might be harder for people working with the UI exclusively to know what col refers to.

Do you have a preference for one type of spelling? If you think col is fairly unambiguous or wordColumnInd isn't that much muddier to read and write I can make it consistent either way. It'd be good to hear your opinion on variable naming, so I can be more well versed in rationales for namings.

Reply:
Agreed! Definitely something to consider, I really don't have a preference on one over the other I personally think `wordColInd` is intuitive enough!

Reply:
@yehee Make sense! I've updated the namings from `.*Column.*` to `.*Col.*` and removed the DOM method in `GoogleSheetsInput.svelte`. I ran uploading via Excel and via Google Sheets to confirm that there were no regressions.

Please let me know if this PR is ready to be merged!